README.rst

```diff
@@ -1,5 +1,6 @@
 |PyPI version| |Docs badge| |Chat badge| |Build Status| |Code Of Conduct| |Mailing Lists| |License|
 
+
```

Comment:
Please remove from this PR, too
```suggestion
```

Reply:
I think i made the changes properly. Let me know if it wasnt